control_panel_api/settings.py

```diff
@@ -122,3 +122,9 @@
 
 LOGIN_URL = 'rest_framework:login'
 LOGOUT_URL = 'rest_framework:logout'
+
+# AWS variables
+BUCKET_REGION = os.environ.get('BUCKET_REGION', 'eu-west-1')
+ENV = os.environ.get('ENV', 'dev')
```

Comment:
FYI: I added these environment variables to the helm chart used to deploy:
 - helm-chart config: https://github.com/ministryofjustice/analytics-platform-config/pull/8
 - helm-chart: https://github.com/ministryofjustice/analytics-platform-helm-charts/pull/16